CONTRIBUTING.md

```diff
@@ -51,6 +51,13 @@ Once the pull request is approved, a team member will take care of merging.
 
 Python 3.7 or later is required.
 
+For Windows, before setting up development environment, make sure you have WSL 
```

Comment:
I would also frame this as a recommendation, not a strict requirement.

For Windows development, we recommend WSL, so you can run the shell scripts in this repository. We will not support Windows Shell/PowerShell. ...